src/hooks/useWebSocket.js

```diff
@@ -4,34 +4,51 @@ import { useStompClient } from 'react-stomp-hooks';
 
 export const useWebSocket = () => {
   const stompClient = useStompClient();
-  const isSubscribedToMessages = useRef(false);
+  const isSubscribedToMessages = useRef(null);
   const isSubscribedToErrors = useRef(false);
 
-  const subscribeToGroupMessages = (endpoint, setCountryData) => {
-    if (stompClient && stompClient.connected && isSubscribedToMessages) {
-      stompClient.subscribe(endpoint, response => {
+  const subscribeToMessages = (endpoint, setCountryData) => {
+    // console.log(`Subscribing to messages at ${endpoint}`);
```

Comment:
Remove commented string before the push your pull request

Reply:
done